src/finch/util/abs_attr.py

```diff
@@ -0,0 +1,66 @@
+class AbsAttr():
```

Comment:
I think in the comment it was `ReqAttrs`?

Reply:
yes, I renamed it